testapi.pm

```diff
@@ -681,8 +660,7 @@ Default timeout is 30s, default stilltime is 7s.
 
 =cut
 
-sub wait_still_screen {
-    my $stilltime = looks_like_number($_[0]) ? shift : 7;
+sub wait_still_screen ($stilltime, @) {
     my $timeout = (@_ % 2) ? shift : $bmwqemu::default_timeout;
```

Comment:
`@_` and `shift`

Reply:
I kept the original code in places without going to full signatures